workers/unity/Assets/Playground/Scripts/MonoBehaviours/FlashOnCollision.cs

```diff
@@ -15,22 +15,22 @@ public class FlashOnCollision : MonoBehaviour
 
     private MeshRenderer renderer;
 
-    private static MaterialPropertyBlock basicMaterial = new MaterialPropertyBlock();
-    private static MaterialPropertyBlock flashingMaterial = new MaterialPropertyBlock();
+    private static MaterialPropertyBlock basicMaterial;
+    private static MaterialPropertyBlock flashingMaterial;
 
     [RuntimeInitializeOnLoadMethod]
     public static void SetupColors()
     {
+        basicMaterial = new MaterialPropertyBlock();
```

Comment:
Just a note, this will probably have some (easy-to-resolve) merge conflicts with my Playground PR. I think your PR is more likely to be merged in first though, so I'll just deal with it when I merge.

Reply:
FYI #259 will also conflict with both